.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
     hooks:
       - id: yamllint
   - repo: https://github.com/awslabs/cfn-python-lint
-    rev: v0.29.0
+    rev: v0.52.0
```

Comment:
There's another workflows issue to update all the pre-commit software, but this one had to be updated now to recognize new Cloudformation types.

Reply:
did that, but just the once to make the updates, not in the build file